src/chains/migrations/0033_walletnew.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 3.2.9 on 2021-11-05 13:15
+from django.apps.registry import Apps
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations import RunPython
+
+
+def copy_wallets(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    Chain = apps.get_model("chains", "Chain")
+    Wallet_Old = apps.get_model("chains", "Wallet")
+    Wallet_New = apps.get_model("chains", "WalletNew")
+
+    # Clears the links between Chain and Wallet
+    for chain in Chain.objects.all():
+        chain.wallet_set.clear()
+
+    for wallet in Wallet_Old.objects.all():
+        Wallet_New.objects.create(name=wallet.name)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("chains", "0032_feature"),
+    ]
+
+    operations = [
+        # Create model with an auto generated id as primary key
+        migrations.CreateModel(
+            name="WalletNew",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "key",
```

Comment:
Why renaming to `key`? I found `name` more self explaining

Reply:
This was a suggestion from Ivan. The reason is that `name` might look like a UI specific field whereas `key` is more of an identifier. We can potentially add name to have a string representation of the `Wallet` in the future. What do you think?

Reply:
No strong opinion, but for me every wallet has a name, nothing related to the UI. Whatever you think is more clear 😃